### ISSUE TITLE
Add own presets / bank into mapped synth.

### ISSUE THREAD

**Issue by TomThak** [jhorology/nks-presets-collection#3]
Hi I just downloaded and tested Spire and it is awesome how it works incl. resources! I also have my banks, that I have bought, is there any possibility to import my own presents / banks into the mapping?

**Comment by jhorology** [comment]
.fxb file ?  I know it's possible to convert programmatically. However I’m not about writing a such program, because Spire is only plugin to use .fxb that I have. Maybe you can find on internet... sorry I don't know. So you may have to do too much bother work.
1.  create nks preset file one by one in komplete kontrol. I recommend to use automation tool such as ["Keyboard Maestro"](http://www.keyboardmaestro.com). Keyboard Masstro [macro file](https://github.com/jhorology/nks-presets-collection/tree/master/src/Spire-1.1/macros) exists in this repository.
2. add metadata(bank/category/author etc)  to NKS file.  [gulp-nks-rewrite-meta](https://www.npmjs.com/package/gulp-nks-rewrite-meta) can do that.
3. replace mapping section of nks file, [gulp-nks-replace-mapping](gulp-nks-replace-mapping) can do that.

**Comment by TomThak** [comment]
Thanks for help!
I tried to copy one of the banks, that you have included and rename the folder and rename the preset name (I just copied one NKS preset and renamed). After rescanning the user content in KK, it still doesnt appear, so I cannot see my own bank folder in Kontrol Komplete. So Do I have to somehow include it in some metafiles? I tried also to rename Factory Bank 6 to for example Factory Bank 77. KK recognized, that Factory Bank 6 no longer exists, but it did not add Factory Bank 77 into the list. How can I include it? I tried to refresh User Content, Factory content and Plug ins in KK so All of files have been refreshed several times, but it still did not appear there. So do I need to follow step 2 and 3 to rewrite something to have there bank folder visibe inside KK?
